Source/WebInspectorUI/UserInterface/Views/SpreadsheetStyleProperty.js

```diff
@@ -368,10 +368,10 @@ WI.SpreadsheetStyleProperty = class SpreadsheetStyleProperty extends WI.Object
         textField.value = isEditingName ? this._property.name : this._property.rawValue;
     }
 
-    spreadsheetTextFieldInitialCompletionIndex(textField, completions)
+    spreadsheetTextFieldInitialCompletionIndex(textField, completions, completionPrefix)
     {
         if (textField === this._nameTextField && WI.settings.experimentalCSSSortPropertyNameAutocompletionByUsage.value)
-            return WI.CSSProperty.indexOfCompletionForMostUsedPropertyName(completions);
+            return WI.CSSProperty.indexOfCompletionForMostUsedPropertyName(completions, completionPrefix);
```

Comment:
I opted to modify `WI.CSSProperty.indexOfCompletionForMostUsedPropertyName` instead of looping again through `completions` to check for an exact match.

Reply:
I know we talked a little about this, but given the function is named `indexOfCompletionForMostUsedPropertyName` I actually feel like we should keep it doing what it's doing and just try to find the exact match here instead. I believe that's a similar amount of work too (like doing two things in one loop, potentially exiting early, is equivalent to doing two loops where the first potentially exits early :D)

Reply:
yeah IMO doing it here would be clearer because the name `indexOfCompletionForMostUsedPropertyName` doesnt imply/indicate anything about a distinction between exact match

tho i can definitely see some value in having the method take into account exact matches as it would allow future callers (if any) to also benefit 🤔

Reply:
`WI.CSSProperty.indexOfCompletionForMostUsedPropertyName` already does two loops through the list of completions: one `Array.prototype.each()` and one `for` loop. I really would like to avoid adding a third one.